keps/sig-testing/2420-reducing-kubernetes-build-maintenance/kep.yml

```
@@ -0,0 +1,46 @@
+title: KEP Template
+kep-number: 2420
+authors:
+  - "BenTheElder"
+owning-sig: sig-testing
+participating-sigs:
+  - sig-release
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+creation-date: yyyy-mm-dd
+reviewers:
+  - dims
+  - liggitt
+approvers:
+  - spiffxp
+  - justaugustus
+prr-approvers:
+  - TBD
+see-also: []
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha|beta|stable
```

Per graduation criteria, we aim to call this stable for v1.21
```suggestion
stage: stable
```